core/lexicon/en/setting.inc.php

```diff
@@ -735,6 +735,9 @@
 $_lang['setting_upload_media'] = 'Uploadable Media Types';
 $_lang['setting_upload_media_desc'] = 'Here you can enter a list of files that can be uploaded into \'assets/media/\' using the Resource Manager. Please enter the extensions for the media types, separated by commas.';
 
+$_lang['setting_upload_translit'] = 'Transliterate names of uploading files?';
+$_lang['setting_upload_translit_desc'] = 'If <b>yes</b> name of any uploading file will be transliterated by global translitiration rules.';
```

Comment:
Here it is worth writing as in the lexicons below, instead of `<b>yes</b>` -> `\'yes\'`

Reply:
Yes, thank you. This fixed